### ISSUE TITLE
Open Collective Roadmap: September 2019 - January 2020

### ISSUE THREAD

**Issue by piamancini** [opencollective/opencollective#2453]
Overview
====== 
The goal of this roadmap is to provide clarity for the core team and our community about our priorities for the next 6 months. While these are the projects we'll be working on, life happens and iterations might take place.

Top level priority for the team is: 
Being [sustainable](https://drive.google.com/drive/u/1/folders/1OwRpuIehFQxRnJIRAksQ1Jd2xXZrhz5L) in the next 12 months while supporting [our mission](https://app.gitbook.com/@opencollective/s/help/) and our community in a way aligned with our [values](https://app.gitbook.com/@opencollective/s/help/the-open-collective-way). 

Roadmap Priorities are aligned with this goal and our framework. 

Prioritization Framework
========

We are a small team and hence we need to work hard to be focused and aligned. When we decide which project / feature to prioritize we judge them against the following questions: 

- **Does this serve our mission? (We are on a mission to help collaborative groups collect and spend money transparently)**
- **Will this make us financially sustainable within a year?**
- Is it feasible to build and maintain with our team and runway?
- Can this make us move faster?
- Does this increase quality and not technical debt?
- Will this bring us more Collective and financial contributors?
- Is this already solved elsewhere or is it our unique contribution?
- Can we measure the success of  this?

New Collective Page
=====
In response to your feedback, we've made some big improvements to the design and functionality of Collective pages (and individual and organization profiles too).
**Owner**: @piamancini 
**Status**: v1 Shipped 09-23-2019
**Metrics**: 
Related complaints and support requests
Fewer people contacting us instead of projects
Conversions to contribution flow
Increasing one-time donations
More custom tiers
How many Collectives use customization options
Page load time

Homepage Redesign
=========
We are re-focusing on projects that want to manage funds through the platform and we are re doing our homepage accordingly. 
**Owner**: @xdamman 
**Status**: In Progress
**Launch**: October
**Metrics**:
Collectives create from homepage
Page load time
Clarity of message (qualitative) - we feel confident about

Onboarding Flow & Business Model
===========
Our goal is to enable communities to become sustainable. We helped raise more than $7M across more than 2,800 collectives hosted by more than 100 different fiscal hosts. Our philosophy has always been to create more value than we capture but if we want to continue being a force for good, we need to make sure that we are ourselves sustainable. 

To refocus the company and to make sure that we can support all those collectives no matter their source of funding, we would love to move to a monthly fee per collective of $20-$10/month. This will also allow us to help collectives get more than just financial contributions

**Issue**: https://github.com/opencollective/opencollective/issues/2366 and https://github.com/opencollective/opencollective/issues/2251
**Owner**: @xdamman 
**Status**: Researching
**Launch**: TBD
**Metrics**: 
More self-hosted and unhosted Collectives created
Increase per-Collective revenue

Expense Flow Redesign
==============
**Issue**: #2394 
**Owner**: @alanna (and @piamancini)
Status: Design 
Implementation: End of October
**Metrics**:
More expenses filed
Less support requests
Less host admin time spent
Way fewer issues with invoices (fewer expense comments)
Use of the new features (ex batch expenses)

Back your Stack
===========
**Owner**: @znarf 
**Status**: Ready to Deploy 
**Metrics**: 
Subscriptions created to Back your Stack.

**Comment by piamancini** [comment]
Post-mortem 
==============

General lessons from projects undertaken this cycle: 
------------------------------------
- We need to better track and follow up with metrics
- We don't iterate enough
- We need a process for amending roadmaps
- Lack of agility (we prioritized conversations a bit quickly)
- Lack of user inputs
- Better scope projects
- Define structural components that engineering can implement while design still iterates. This is about separating the individual components from the layout in design. Layout will always change but components should not evolve that much.
- We have much higher quality, consistency
- Setting time constraints
- Better time projects
- Set expectations right
- Defined feedback moments 
- We’ve become better at reacting

New Collective Page
=============
**Positive:**
Good result
Code quality
[Meta] Helped improve our processes
Project ownership
**Negative**:
Took too long
Mutated a lot
No iterations after shipping
Not enough user enquiry?
Scope was missing

Conversations
============
**Positive**:
Fast design
Consistent
Efficient

**Negative**:
Not on the roadmap
Lack of iteration
Lack of user insights

Homepage
==========
**Positive**:
External designed, working in the open
Defining graphic language

**Negative**:
1 designer for 2 projects in parallel 
Illustrations are time consuming
Went out of scope (we defined a new graphic language)
Process was too long and too complex for a static page

New business model (host plans)
===============
**Positive**:
Independant
It happened fast
Conversation with the users worked great
Full MVP (didn’t take too much engineering time)
We reacted well 

**Negative**:
Originally bundled with Create Collective Flow
Too hacky originally
Lack of metrics & revenue projections

Create collective flow
==========
**Positive**:
Designer worked well
Good result
Feedback altering structure

**Negative**:
Design team bottleneck after initial design
No brief for Karim to start
Scope (put together with the new business model)

Expense flow
=============
**Positive**:
Well scoped, realistic
Tons of input/insights, issues well documented
Simple, looks good
Split in multiple independent chunks, that made  it easy to work on for design
Several well organized feedback rounds

**Negative**:
Given the right resources, it could have started earlier
3 months to get the design done without starting implementation ⇒ 6 months project
Assess better / / focus

BackYourStack
=============
**Positive**:
MVP
Good design
Changed owner (Alyssa)

**Negative**:
Lack of clarity about being co-owners
Implementation should have taken less time
Maybe not the right person to work on this project
Shipped as an MVP
Missed metrics
Lack of consistent weekly progress.
